LPS-123609 portal-search-test: Test for FacetedSearcher

brandizzi opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

brandizzi commented:
Tests failure output below. Apparently, disabling the clauses make a query a "match_all" query:

```
> Task :apps:portal-search:portal-search-test:testIntegration

com.liferay.portal.search.searcher.test.IndexerClausesSuppressionTest STANDARD_OUT
    Loading jar:file:/home/adam/lib/liferay/liferay-portal-master/tomcat-9.0.37/webapps/ROOT/WEB-INF/lib/portal-impl.jar!/system.properties
    Loading jar:file:/home/adam/software/liferay-portal/.m2/com/liferay/portal/com.liferay.portal.impl/6.0.0-SNAPSHOT/com.liferay.portal.impl-6.0.0-SNAPSHOT.jar!/system.properties

com.liferay.portal.search.searcher.test.IndexerClausesSuppressionTest > testSuppressClauseWithBaseIndexer STARTED

com.liferay.portal.search.searcher.test.IndexerClausesSuppressionTest > testSuppressClauseWithBaseIndexer FAILED
    org.junit.ComparisonFailure: {"from":0,"size":10000,"query":{"bool":{"must":[{"match_all":{"boost":1.0}}],"filter":[{"bool":{"must":[{"bool":{"must":[{"terms":{"groupId":["43643"]}},{"terms":{"scopeGroupId":["43643"]}}]}},{"bool":{"should":[{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.journal.model.JournalArticle"}}}]}}]}}]}}]}},"explain":false,"stored_fields":"*","track_scores":true,"track_total_hits":2147483647}->{ddmTemplateKey=BASIC-WEB-CONTENT, entryClassPK=43648, localized_title_sv_SE=gamma, groupId=43643, publishDate=20210122202700, classNameId=0, createDate_sortable=1611347260935, uuid=e5d017ec-27a5-a69b-9c09-a07679542e37, localized_title_nl_NL=gamma, classPK=0, localized_title_zh_CN=gamma, scopeGroupId=43643, articleId_String_sortable=43646, modified=20210122202741, localized_title_ar_SA_sortable=gamma, modified_sortable=1611347261392, localized_title_pt_BR=gamma, localized_title_ca_ES_sortable=gamma, content_en_US=alpha, localized_title_zh_CN_sortable=gamma, localized_title_ca_ES=gamma, localized_title_hu_HU_sortable=gamma, visible=true, ddmStructureKey=BASIC-WEB-CONTENT, localized_title_nl_NL_sortable=gamma, defaultLanguageId=en_US, viewCount_sortable=0, priority=0.0, version=1.0, folderId=0, classTypeId=29801, companyId=20101, rootEntryClassPK=43648, displayDate=20210122202700, localized_title_fr_FR=gamma, title_en_US=gamma, status=0, localized_title_hu_HU=gamma, localized_title_pt_BR_sortable=gamma, publishDate_sortable=1611347220000, localized_title_fi_FI_sortable=gamma, head=true, uid=com.liferay.journal.model.JournalArticle_PORTLET_43647, localized_title_de_DE_sortable=gamma, localized_title_en_US_sortable=gamma, groupRoleId=43643-20118, localized_title_fr_FR_sortable=gamma, layoutUuid=, urlTitle_en_US=gamma, expirationDate_sortable=9223372036854775807, viewCount=0, localized_title_sv_SE_sortable=gamma, localized_title_ja_JP=gamma, localized_title_fi_FI=gamma, createDate=20210122202740, expirationDate=99950812133000, latest=true, displayDate_sortable=1611347220000, localized_title_es_ES_sortable=gamma, entryClassName=com.liferay.journal.model.JournalArticle, roleId=[20110, 20111], articleId=43646, userName=test test, userId=20129, urlTitle_en_US_String_sortable=gamma, localized_title_ja_JP_sortable=gamma, localized_title=gamma, stagingGroup=false, headListable=true, localized_title_es_ES=gamma, localized_title_en_US=gamma, localized_title_ar_SA=gamma, treePath=, localized_title_de_DE=gamma}->[gamma] expected:<[[]]> but was:<[[gamma]]>
        at org.junit.Assert.assertEquals(Assert.java:117)
        at com.liferay.portal.search.test.util.DocumentsAssert.assertValuesIgnoreRelevance(DocumentsAssert.java:102)
        at com.liferay.portal.search.searcher.test.IndexerClausesSuppressionTest.assertSearch(IndexerClausesSuppressionTest.java:272)
        at com.liferay.portal.search.searcher.test.IndexerClausesSuppressionTest.testSuppressClauseWithBaseIndexer(IndexerClausesSuppressionTest.java:126)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$3.evaluate(TestExecutorRunnable.java:353)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
        at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
        at org.junit.rules.RunRules.evaluate(RunRules.java:20)
        at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$1.evaluate(TestExecutorRunnable.java:237)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at org.junit.rules.RunRules.evaluate(RunRules.java:20)
        at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable._execute(TestExecutorRunnable.java:316)
        at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable.run(TestExecutorRunnable.java:95)
        at java.base/java.lang.Thread.run(Thread.java:834)

com.liferay.portal.search.searcher.test.IndexerClausesSuppressionTest > testSuppressClauseWithDefaultIndexer STARTED

com.liferay.portal.search.searcher.test.IndexerClausesSuppressionTest > testSuppressClauseWithDefaultIndexer FAILED
    org.junit.ComparisonFailure: {"from":0,"size":10000,"query":{"bool":{"must":[{"bool":{"must":[{"bool":{"should":[{"bool":{"must":[{"match":{"city":{"query":"omega"}}}],"should":[{"match_phrase":{"city":{"query":"omega","slop":50}}},{"match_phrase":{"city":{"query":"omega","boost":2.0}}}]}},{"bool":{"must":[{"match":{"country":{"query":"omega"}}}],"should":[{"match_phrase":{"country":{"query":"omega","slop":50}}},{"match_phrase":{"country":{"query":"omega","boost":2.0}}}]}},{"bool":{"should":[{"wildcard":{"emailAddress":{"wildcard":"*omega*"}}}]}},{"bool":{"must":[{"match":{"firstName":{"query":"omega"}}}],"should":[{"match_phrase":{"firstName":{"query":"omega","slop":50}}},{"match_phrase":{"firstName":{"query":"omega","boost":2.0}}}]}},{"bool":{"must":[{"match":{"fullName":{"query":"omega"}}}],"should":[{"match_phrase":{"fullName":{"query":"omega","slop":50}}},{"match_phrase":{"fullName":{"query":"omega","boost":2.0}}}]}},{"bool":{"must":[{"match":{"jobTitle":{"query":"omega"}}}],"should":[{"match_phrase":{"jobTitle":{"query":"omega","slop":50}}},{"match_phrase":{"jobTitle":{"query":"omega","boost":2.0}}}]}},{"bool":{"must":[{"match":{"lastName":{"query":"omega"}}}],"should":[{"match_phrase":{"lastName":{"query":"omega","slop":50}}},{"match_phrase":{"lastName":{"query":"omega","boost":2.0}}}]}},{"bool":{"must":[{"match":{"middleName":{"query":"omega"}}}],"should":[{"match_phrase":{"middleName":{"query":"omega","slop":50}}},{"match_phrase":{"middleName":{"query":"omega","boost":2.0}}}]}},{"bool":{"must":[{"match":{"region":{"query":"omega"}}}],"should":[{"match_phrase":{"region":{"query":"omega","slop":50}}},{"match_phrase":{"region":{"query":"omega","boost":2.0}}}]}},{"bool":{"should":[{"wildcard":{"screenName":{"wildcard":"*omega*"}}}]}},{"bool":{"must":[{"match":{"street":{"query":"omega"}}}],"should":[{"match_phrase":{"street":{"query":"omega","slop":50}}},{"match_phrase":{"street":{"query":"omega","boost":2.0}}}]}},{"bool":{"must":[{"match":{"zip":{"query":"omega"}}}],"should":[{"match_phrase":{"zip":{"query":"omega","slop":50}}},{"match_phrase":{"zip":{"query":"omega","boost":2.0}}}]}}]}}]}}],"filter":[{"bool":{"must":[{"bool":{"must":[{"terms":{"groupId":["43661"]}},{"terms":{"scopeGroupId":["43661"]}}]}},{"bool":{"should":[{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.portal.kernel.model.User"}}}]}}]}}]}}]}},"explain":false,"stored_fields":"*","track_scores":true,"track_total_hits":2147483647}->{lastName=omega, entryClassPK=43665, firstName_sortable=gama, groupId=43661, screenName=alpha, uid=com.liferay.portal.kernel.model.User_PORTLET_43665, emailAddress=ce5kc4kp43664@liferay.com, scopeGroupId=43661, groupIds=43661, modified=20210122202745, defaultUser=false, modified_sortable=1611347265728, birthDate_sortable=0, entryClassName=com.liferay.portal.kernel.model.User, roleId=20111, fullName=gama omega, userName=gama omega, userId=43665, birthDate=19700101000000, organizationCount=0, firstName=gama, screenName_sortable=alpha, companyId=20101, roleIds=20113, lastName_sortable=omega, status=0}->[gama omega] expected:<[[]]> but was:<[[gama omega]]>
        at org.junit.Assert.assertEquals(Assert.java:117)
        at com.liferay.portal.search.test.util.DocumentsAssert.assertValuesIgnoreRelevance(DocumentsAssert.java:102)
        at com.liferay.portal.search.searcher.test.IndexerClausesSuppressionTest.assertSearch(IndexerClausesSuppressionTest.java:272)
        at com.liferay.portal.search.searcher.test.IndexerClausesSuppressionTest.testSuppressClauseWithDefaultIndexer(IndexerClausesSuppressionTest.java:162)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$3.evaluate(TestExecutorRunnable.java:353)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
        at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
        at org.junit.rules.RunRules.evaluate(RunRules.java:20)
        at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$1.evaluate(TestExecutorRunnable.java:237)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at org.junit.rules.RunRules.evaluate(RunRules.java:20)
        at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable._execute(TestExecutorRunnable.java:316)
        at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable.run(TestExecutorRunnable.java:95)
        at java.base/java.lang.Thread.run(Thread.java:834)

com.liferay.portal.search.searcher.test.IndexerClausesSuppressionTest > testSuppressClauseWithFacetedSearcher STARTED

com.liferay.portal.search.searcher.test.IndexerClausesSuppressionTest > testSuppressClauseWithFacetedSearcher FAILED
    org.junit.ComparisonFailure: {"from":0,"size":10000,"query":{"bool":{"must":[{"match_all":{"boost":1.0}}],"filter":[{"bool":{"must":[{"term":{"companyId":{"value":"20101"}}},{"bool":{"must":[{"terms":{"groupId":["43681"]}},{"terms":{"scopeGroupId":["43681"]}}]}},{"bool":{"should":[{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.wiki.model.WikiPage"}}}]}},{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.commerce.product.model.CPDefinition"}}}]}},{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.blogs.model.BlogsEntry"}}}]}},{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.dynamic.data.mapping.model.DDMFormInstanceRecord"}}}]}},{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.bookmarks.model.BookmarksEntry"}}}]}},{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.journal.model.JournalArticle"}}}]}},{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.journal.model.JournalFolder"}}}]}},{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.calendar.model.CalendarBooking"}}}]}},{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.document.library.kernel.model.DLFileEntry"}}}]}},{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.portal.kernel.model.User"}}}]}},{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.bookmarks.model.BookmarksFolder"}}}]}},{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.portal.kernel.model.Layout"}}}]}},{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.document.library.kernel.model.DLFolder"}}}]}},{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.dynamic.data.lists.model.DDLRecord"}}}]}},{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.message.boards.model.MBMessage"}}}]}},{"bool":{"must":[{"term":{"entryClassName":{"value":"com.liferay.knowledge.base.model.KBArticle"}}}]}}]}}]}}]}},"explain":false,"stored_fields":"*","track_scores":true,"track_total_hits":2147483647}->{lastName=Test, entryClassPK=20129, firstName_sortable=test, groupId=[43681, 20125, 20154], screenName=test, uid=com.liferay.portal.kernel.model.User_PORTLET_20129, emailAddress=test@liferay.com, scopeGroupId=[43681, 20125, 20154], organizationIds=20153, groupIds=[20125, 43681], modified=20210121142727, defaultUser=false, modified_sortable=1611239247739, birthDate_sortable=0, entryClassName=com.liferay.portal.kernel.model.User, roleId=20111, fullName=Test Test, userName=test test, userId=20129, birthDate=19700101000000, organizationCount=1, firstName=Test, screenName_sortable=test, companyId=20101, roleIds=[20108, 20112, 20113], lastName_sortable=test, status=0},{lastName=omega, entryClassPK=43685, firstName_sortable=gama, groupId=43681, screenName=alpha, uid=com.liferay.portal.kernel.model.User_PORTLET_43685, emailAddress=bi5huqfp43684@liferay.com, scopeGroupId=43681, groupIds=43681, modified=20210122202748, defaultUser=false, modified_sortable=1611347268354, birthDate_sortable=0, entryClassName=com.liferay.portal.kernel.model.User, roleId=20111, fullName=gama omega, userName=gama omega, userId=43685, birthDate=19700101000000, organizationCount=0, firstName=gama, screenName_sortable=alpha, companyId=20101, roleIds=20113, lastName_sortable=omega, status=0},{ddmTemplateKey=BASIC-WEB-CONTENT, entryClassPK=43696, localized_title_sv_SE=omega, groupId=43681, publishDate=20210122202700, classNameId=0, createDate_sortable=1611347268462, uuid=1b0464d9-8bbd-1f0a-45ef-75520ee3a7d8, localized_title_nl_NL=omega, classPK=0, localized_title_zh_CN=omega, scopeGroupId=43681, articleId_String_sortable=43694, modified=20210122202748, localized_title_ar_SA_sortable=omega, modified_sortable=1611347268548, localized_title_pt_BR=omega, localized_title_ca_ES_sortable=omega, content_en_US=alpha, localized_title_zh_CN_sortable=omega, localized_title_ca_ES=omega, localized_title_hu_HU_sortable=omega, visible=true, ddmStructureKey=BASIC-WEB-CONTENT, localized_title_nl_NL_sortable=omega, defaultLanguageId=en_US, viewCount_sortable=0, priority=
        at org.junit.Assert.assertEquals(Assert.java:117)
        at com.liferay.portal.search.test.util.DocumentsAssert.assertValuesIgnoreRelevance(DocumentsAssert.java:102)
        at com.liferay.portal.search.searcher.test.IndexerClausesSuppressionTest.assertSearch(IndexerClausesSuppressionTest.java:272)
        at com.liferay.portal.search.searcher.test.IndexerClausesSuppressionTest.testSuppressClauseWithFacetedSearcher(IndexerClausesSuppressionTest.java:201)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$3.evaluate(TestExecutorRunnable.java:353)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
        at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
        at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
        at org.junit.rules.RunRules.evaluate(RunRules.java:20)
        at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$1.evaluate(TestExecutorRunnable.java:237)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at com.liferay.portal.kernel.test.rule.AbstractTestRule$1.evaluate(AbstractTestRule.java:59)
        at org.junit.rules.RunRules.evaluate(RunRules.java:20)
        at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable._execute(TestExecutorRunnable.java:316)
        at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable.run(TestExecutorRunnable.java:95)
        at java.base/java.lang.Thread.run(Thread.java:834)

3 tests completed, 3 failed
```
